scripts/build_curl_openssl.sh

```diff
@@ -91,3 +92,31 @@ PKG_CONFIG_PATH="$OPENSSL_BUILD_DIR/lib/pkgconfig" LIBS="-ldl" ./configure ${cur
 echo "Building and Installing Curl"
 make 2>&1 > /dev/null
 make install 2>&1 > /dev/null
+
+# build aws-cpp-sdk
```

Comment:
As we talked in the meeting, can you create a separate script for AWS SDK build so that the build_curl_openssl.sh calls? It would be great if you change build_curl_openssl.sh to build_dependencies.sh as well.